src/elements/call-out/src/index.tsx

```diff
@@ -38,13 +51,22 @@ const CallOut: React.FC<Props> = ({ title, text, icon }) => {
         marginBottom: 'md',
         variant: 'callOut.main'
       }}
+      px={{
+        backgroundColor: 'accentColor'
+      }}
     >
       {icon ? (
         <div sx={{ display: 'flex' }}>
-          <div sx={{ flex: '0 0 16px', marginRight: '8px' }}>
+          <div
+            sx={{
+              flexGrow: 0,
+              flexShrink: 0,
+              flexBasis: ({ space }) => space.sm,
```

Comment:
🤯 I didn't realise you could use function syntax on individual properties! Awesome :)

Reply:
Yeah, each property is supplied the `theme` as the first argument is the value is a function 😄